app/_includes/prereqs/install/fips/red-hat.md

```diff
@@ -0,0 +1,33 @@
+{% navtabs %}
+{% navtab "Manual Installation" %}
+1. Download {{site.base_gateway}}:
+    ```sh
```

Comment:
Remove the indents in these codeblocks (see my slack message) so that we don't get extra spaces at the start of the codeblock.